source/includes/pipeline_config/_005-svn-material.md.erb

```diff
@@ -0,0 +1,31 @@
+<%
+json ='{
+  "materials": [
+    {
+      "type": "svn",
+      "attributes": {
+        "url": "svn://my_ip//repository//project//trunk",
```

Comment:
Is that even a legit URL, how about `svn://svn.example.com/myProject/trunk` (or something along those lines)?